pkg/resource/acl/sdk.go

```diff
@@ -164,12 +164,14 @@ func (rm *resourceManager) sdkFind(
 	}
 
 	rm.setStatusDefaults(ko)
-	resourceARN := (*string)(ko.Status.ACKResourceMetadata.ARN)
-	tags, err := rm.getTags(ctx, *resourceARN)
-	if err != nil {
-		return nil, err
+	if rm.aclActive(&resource{ko}) {
```

Comment:
If not active, lets set it to `resourceSynced = false` condition so that it continues to requeue until it is active.

Reply:
We cannot, it causes a bug.

Reply:
It causes failure for all snapshot and cluster creations.

Reply:
Nevermind, you already have the logic for using the status to determine `IsSynced` in your `generator.yaml`. No need to write custom code when we already support that. Great